frontend/src/component/menu/Header/NavigationMenu/NavigationMenu.tsx

```diff
@@ -62,16 +79,13 @@ export const NavigationMenu = ({
         >
             {options
                 .map((option, i) => [
-                    <ConditionallyRender
-                        key={`${option.path}-divider`}
-                        condition={Boolean(
-                            showUpdatedMenu &&
-                                options[i - 1]?.group &&
-                                options[i - 1]?.group !== option.group
-                        )}
-                        show={<Divider variant="middle" />}
-                        elseShow={null}
-                    />,
+                    Boolean(
+                        showUpdatedMenu &&
+                            options[i - 1]?.group &&
```

Comment:
not very clear what those indexes are. I'd use some intention revealing variable/fn here

Reply:
refactored to variable `previousGroup`